WooCommerce/WooCommerceUITests/Tests/PaymentsTests.swift

```diff
@@ -22,7 +22,17 @@ final class PaymentsTests: XCTestCase {
     }
 
     func test_load_learn_more_link() throws {
-        try PaymentsScreen().tapLearnMoreIPPLink()
+        guard UIDevice.current.userInterfaceIdiom == .pad else {
+            throw XCTSkip(
+                """
+                Skipping on the iPhone.
+                Calling tap() on a link within attributed text no longer works in UI tests on Xcode 16.
+                Using click() instead, which is only supported on iPad.
+                """
+            )
+        }
```

Comment:
What do you think of moving this check in the `click` method os that every caller gets it ? 

I realize there might not be other callers ever, but it seems structurally better.

Reply:
Yes, it makes sense. iPhone test will test at least part of the flow.